cmd/wallet-web/src/pages/Credentials.vue

```diff
@@ -32,17 +32,15 @@
         <flyout-menu />
       </div>
     </div>
-    <skeleton-loader v-if="loadingStatus === 'inProgress'" type="vault" />
+    <skeleton-loader v-if="loadingStatus === 'inProgress' && loading" type="vault" />
```

Comment:
Why we need this loading check ?

Reply:
We should be displaying the loading state at first, until the all credentials and vault names are fetched. See the video in the PR description for reference.